src/logic/wallets/ethTransactions.ts

```diff
@@ -61,7 +61,7 @@ export const calculateGasOf = async (txConfig: {
   try {
     const gas = await web3.eth.estimateGas(txConfig)
 
-    return gas * 2
+    return gas
```

Comment:
I think we don't need anymore this code: 
```
if (process.env.NODE_ENV === 'test') {
    return '20000000000'
}
  ```

Reply:
Done